scripts/ci/update_server_names.py

```diff
@@ -0,0 +1,156 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import re
+import requests
+import sys
+import xml.etree.ElementTree as etree
+from translate.misc.xml_helpers import reindent
+
+MULLVAD_SERVER_LIST_API = "https://api.mullvad.net/public/relays/wireguard/v2"
+
+def fetch_server_list():
+    country_names = []
+    city_names = []
+
+    try:
+        response = requests.get(MULLVAD_SERVER_LIST_API)
+
+        if response.status_code == 200:
+            data = response.json()
+
+            locations = data.get('locations', None)
```

Comment:
```suggestion
            locations = data.get('locations', {})
```

Set the fallback to an empty dictionary and drop the `if locations`.